src/store/integrations/composer.tsx

```diff
@@ -57,6 +57,7 @@ type ComposerProps = EditorProps & {
 	 * If defined, a menu item to add inline images is added to the composer.
 	 */
 	onFileSelect?: (arg: { editor: TinyMCE; files: HTMLInputElement['files'] | undefined }) => void;
+	customInitOptions?: EditorProps['init'];
```

Comment:
I think it could be safer if the type was marked as Partial, even if at the moment all fields are optional inside that init field of the EditorProps
```suggestion
	customInitOptions?: Partial<EditorProps['init']>;
```